tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,10 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result == False, f"No item in inventory expected to be False, but got {result}"
```

Comment:
Nice work. A couple of things that you could add is an assert statement that checks to see if `item` is **not** inside of the inventory as well as statements checking to see if the original items are in the inventory as well. 
```suggestion
    assert result == False, f"No item in inventory expected to be False, but got {result}"
    assert "a" in vendor.inventory
    assert "b" in vendor.inventory
    assert "c" in vendor.inventory
    assert item not in vendor.inventory
```